index.js

```diff
@@ -44,8 +44,18 @@ const renderHelper = (tasks, options, level) => {
 	return output.join('\n');
 };
 
+const shouldSuspendUpdateRenderer = (tasks) => {
+
+	for(const task of tasks){
+		if(task.isEnabled() && task.isPending() && !task.isCompleted()){
+			return task.shouldSuspendUpdateRenderer();
+		}
+	}
+}
```

Comment:
I feel like this logic belongs in `listr` and not the renderer.

Reply:
After writing the tests, i remove it to `listr`.

Reply:
This logic has been removed to listr with the commit [4685274](https://github.com/SamVerschueren/listr-update-renderer/pull/23/commits/4685274ffa8c529d5a404ad89d778c3a9f7b56a9).